README.md

```diff
@@ -41,6 +41,23 @@ TODO
 * Full usage for each resource
 * Proper explanation of the `AfterbanksPSD2:Error` and its subclasses
 
+List of Error
+------------
+
+ *    `1` -> GenericError
+ *   `50` -> IncorrectParametersError (When call API)
+ * `C000` -> GenericConsentError
+ * `C001` -> InvalidConsentError
+ * `C002` -> ConsentWithUnfinalizedProcessError
+ * `C003` -> ProductMismatchConsentError
+ * `C004` -> ExpiredConsentError
+ * `C005` -> MaximumNumberOfCallsReachedConsentError
+ * `T000` -> GenericTransactionError
+ * `T001` -> InvalidConsentForProductError
+ * `P000` -> Error genérico con la iniciación de pagos.
```

Comment:
Doesn't this one have its own class as the others?

Reply:
I will remove this code error in the Readme, `PXXX` are the errors for the payment function.
We don't use this part.